shared/src/hooks/useFetchSubgraphData.ts

```diff
@@ -65,10 +66,16 @@ const useFetchSubgraphData = () => {
 
     const allSubgraphResponses = await Promise.all(
       SUBGRAPHS_TO_QUERY.map(async ([version, chainId]) => {
-        const response = await axios.post(
-          getSubgraphURIForVersion(version, chainId),
-          {
-            query: `{
+        let query;
+
+        switch (version) {
+          case "governance":
```

Comment:
This is a confusing switch because `version` here refers to v1 or v2 vaults. It's worth it to just split this out into a different hook like `useFetchGovSubgraph`